### ISSUE TITLE
Need to prompt user for name

### ISSUE THREAD

**Issue by TildeWilliams** [plus2/whereuat#5]
Since the same API key is always used, it'd be great if the person had to give their name when rejecting the story so I know who to follow up with. Alternately maybe we can present them with a drop down of the members on the project? Either way, just concat that value to the beginning or end of the rejection message.

**Comment by jondkinney** [comment]
I agree with this need. Or allow them to login to pivotal and pull that user's auth token or something.
